Update HubSpot Source to add custom property instructions (#4715)
segmentio/segment-docs: src/connections/sources/catalog/cloud-apps/hubspot/index.md

Below are tables outlining the properties included in the collections listed above. To see the full description of each property, refer to the HubSpot documentation linked in the collections above.

> If you have Custom Properties on Contacts or Companies collections that you would like to sync, submit a ticket detailing the custom properties [here](http://segment.com/help/contact) and we can enable it for you. 

Reviewer: ```suggestion
If you have Custom Properties on Contacts or Companies collections that you would like to sync, submit a ticket detailing the custom properties [here](http://segment.com/help/contact){:target="_blank"}.
```

Below are tables outlining the properties included in the collections listed above. To see the full description of each property, refer to the HubSpot documentation linked in the collections above.

> If you have Custom Properties on Contacts or Companies collections that you would like to sync, submit a ticket detailing the custom properties [here](http://segment.com/help/contact) and we can enable it for you. **Note:** For Deals collection, Segment pulls whatever properties the HubSpot API returns, which means you can add the new fields on your own from HubSpot if you have the necessary permissions.

Reviewer: ```suggestion
If you have Custom Properties on Contacts or Companies collections that you would like to sync, submit a ticket detailing the custom properties [here](http://segment.com/help/contact) and we can enable it for you. 

> info ""
> For Deals collection, Segment retrieves properties that the HubSpot API returns, which means you can add the new fields on your own from HubSpot if you have the necessary permissions.
```
